CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 ### Changes
 
+- Added an OIDC AllowGroups Configuration options and authorization check [#1041](https://github.com/juanfont/headscale/pull/1041)
```

Comment:
Good, could you just move it up to 0.18.x?

Reply:
You got it! I just amended the commit.